buildSrc/src/main/groovy/fahrschein.java-conventions.gradle

```diff
@@ -0,0 +1,60 @@
+plugins {
+    id 'java-library'
+    id 'jacoco'
+    id 'org.owasp.dependencycheck'
+}
+
+repositories {
+    mavenCentral()
+}
+
+group = 'org.zalando'
+version = project.property('project.version') + (project.hasProperty('release') ? '' : '-SNAPSHOT')
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+java.targetCompatibility = JavaVersion.VERSION_1_8
+
+tasks.withType(JavaCompile).configureEach {
```

Comment:
You have a configuration block for `JavaCompile` here and at the end. You should be able to merge them into one, no?!